sdcm/tester.py

```diff
@@ -1010,10 +1010,11 @@ def get_cluster_k8s_gke(self):
         # NOTE: between GKE cluster creation and addition of new node pools we need
         # several minutes gap to avoid "repair" status of a cluster when API server goes down.
         # So, deploy apps specific to default-pool in between above mentioned deployment steps.
-        self.k8s_cluster.deploy_cert_manager()
+        self.k8s_cluster.set_nodeselector_for_kubedns()
```

Comment:
It would be great to be consistent and explicitly pass pool_name here too
Also If we stick to this approach having default value for pool_name attribute in set_nodeselector_for_kubedns have no sense.

Reply:
Done